docs/user/01-10-kyma-crd.md

```diff
@@ -0,0 +1,11 @@
+# Kyma
```

Comment:
In contributors folder the CRD files don't have a `-crd` suffix. Intended?

Reply:
It was... as in the `contributor` they are in a directory called resources so we know where the files belong to and what they are about. But in the `user` it is the only doc now and there is no context. But I'm starting having second thoughts so we might want to change that later.

Reply:
But makes sense for me then. As long as there is a reasoning behind it I am not dogmatic about it :)